content/docs/ref/ci.md

```diff
@@ -26,5 +25,5 @@ Instead of wrangling with
 simply run:
 
 ```cli
-$ cml ci
+$ cml ci --fetch-depth=0
```

Comment:
Product Q: Should it be the default value?

Reply:
There is no default value for this option, Imagine that 95% of the use cases will be `--fetch-depth=0` or the previous alias for this `--unshallow`

Reply:
`--fetch-depth=0` is slow & space-consuming so can't be the default. Also, we can't use [--fetch-depth=1 imitating GHA](https://github.com/actions/checkout/blob/1f9a0c22da41e6ebfa534300ef656657ea2c6707/action.yml#L58) because that would truncate unshallow repos.